app/controllers/TarRules.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import engine.Condition._
+import engine.RuleEngine
+import model.Locations._
+import model._
+
+object TarRules extends RuleEngine {
+
+  override val rules = List(
+    when(LoggedInViaVerify and not(ArrivingWithCredId)) thenGoTo PersonalTaxAccount withName "pta-home-page-for-verify-user",
```

Comment:
Agreed `ArrivingWithCredId` can be combined with `LoggedInViaGovernmentGateway` in `IsGovernmentGatewayUser`